core/storage/config/gae_models_test.py

```diff
@@ -194,3 +210,21 @@ def test_commit_with_updated_rules(self) -> None:
         assert retrieved_model is not None
 
         self.assertEqual(retrieved_model.rules, new_rules)
+
+    def test_get_model_association_to_user(self) -> None:
+        self.assertEqual(
+            config_models.PlatformParameterModel.
+            get_model_association_to_user(),
```

Comment:
In Python we try to avoid breaking on `.`. You might be able to avoid it with something like this:

```python
association = config_models.PlatformParameterModel.get_model_association_to_user(),
self.assertEqual(
    association,
    base_models.MODEL_ASSOCIATION_TO_USER.NOT_CORRESPONDING_TO_USER
)
```

Alternatively, you can indent this line to make this easier to read

Reply:
Done!